examples/google-esp-integration/server_only/views/header.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8" />
+  <title>UID2 Publisher Standard Integration Example</title>
+  <link rel="stylesheet" type="text/css" href="/stylesheets/app.css" />
+  <link rel="shortcut icon" href="/images/favicon.png" />
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script async src="/js/mockedGoogleTag.js"></script>
+  <script async src="/js/uid2Esp.js"></script>
+  <script type="text/javascript">
+    const getCookie = (cname) => {
+      let name = cname + "=";
+      let ca = document.cookie.split(';');
+      for(let i = 0; i < ca.length; i++) {
+        let c = ca[i];
+        while (c.charAt(0) == ' ') {
+          c = c.substring(1);
+        }
+        if (c.indexOf(name) == 0) {
+          return c.substring(name.length, c.length);
+        }
+      }
+      return "";
+    }
+
+    const getAdvertisingToken = () => {
+      const uid2Token = getCookie('identity')
+      if (uid2Token === "") return
+      return JSON.parse(decodeURIComponent(uid2Token)).advertising_token
+    }
+
+    window.getAdvertisingToken = getAdvertisingToken;
+  
+    $(document).ready(function() {
+      if(window.location.href.endsWith('/login')){
+        window.googletag.cmd.push(window.googletag.encryptedSignalProviders.clearAllCache)
```

Comment:
I think `encryptedSignalProviders` is deprecated -- should we switch to `secureSignalProviders` everywhere?